airgun/views/hostcollection.py

```diff
@@ -188,7 +188,7 @@ def get_action_button(self, name):
             )
         return action_button
 
-    def apply_action(self, name, action_via='via Katello Agent'):
+    def apply_action(self, name, action_via='via remote execution'):
```

Comment:
same here

Reply:
same as above - we still have two methods and we pick one of them